schema/filter_artworks.js

```diff
@@ -186,6 +216,7 @@ function filterArtworks(primaryKey) {
       }
 
       return gravity.with(accessToken)("filter/artworks", gravityOptions)
+        .then(response => assign({}, response, { options: gravityOptions }))
```

Comment:
@mzikherman Yeah exactly, this is mostly to do with the fact that it would be awkward to make a component on the react-relay side with a dynamic query. 

Consider this (old) code: https://github.com/artsy/force/blob/master/desktop/components/filter2/headline/view.coffee#L40

To accomplish this same thing with GraphQL + React / Relay we could approach it a few different ways: 
1. Like you said, we could pull it from the params, but then we have to make a query like so:
```
     fragment on Viewer {
        filter_artworks(
          aggregations: $aggregations, 
          size: $size,
          for_sale: $for_sale,
          medium: $medium,
          price_range: $price_range,
          dimension_range: $dimension_range,
          sort: $sort,
          gene_id: $gene_id,
          tag_id: $tag_id,
          artist_id: $artist_id,
        ) {
          ${TotalCount.getFragment("filter_artworks")}
          aggregations {
            ${Dropdown.getFragment("aggregation")}
          }
          artworks: artworks_connection(first: $size) {
            ${Artworks.getFragment("artworks")}
          }
          tag(id: $tag_id) @include(if:$tag_id){
             name
          }
          gene(id: $gene_id) @include(if:$gene_id){
             name
          }
          // etc
        }
      }
```
This is _OK_, but IMO it is more awkward to query for these objects outside the context of the `Headline` component (since nothing else needs to know about it). We have to check which object is present, outside the context of the headline, etc.

2. We could take my approach and let metaphysics determine what the focus of the filtered search is (like I said, the cases where we do this currently are for Genes, Tags, and Artists). This way, on the reaction-force side, all we have to do is call `facet.name`. 

Happy to take either approach here (or possibly there is a more elegant way that is escaping me).